### ISSUE TITLE
Android fatal runtime error when not initialized but bluetooth state changes

### ISSUE THREAD

**Issue by cubedude** [rohitsangwan01/ble_peripheral#10]
So i stumbled upon a curious bug, like the title said.

The workflow is currently this:

1: Permissions need to be asked and given, that means the user has some time on page if they refuse the first time around.
2: Then and only then can BlePeripheral.initialize() be called.
Now, if the user  declines the permissions, initialize hasn't been called, and the user manually turns bluetooth on or off (or its done by some other way), the whole application will fatal crash and close with the stacktrace:

```
java.lang.RuntimeException: Error receiving broadcast Intent { act=android.bluetooth.adapter.action.STATE_CHANGED flg=0x4000010 (has extras) } in com.rohit.ble_peripheral.BlePeripheralPlugin$broadcastReceiver$1@f88b5e9
	at android.app.LoadedApk$ReceiverDispatcher$Args.lambda$getRunnable$0(LoadedApk.java:1810)
	at android.app.LoadedApk$ReceiverDispatcher$Args.$r8$lambda$gDuJqgxY6Zb-ifyeubKeivTLAwk(Unknown Source:0)
	at android.app.LoadedApk$ReceiverDispatcher$Args$$ExternalSyntheticLambda0.run(Unknown Source:2)
	at android.os.Handler.handleCallback(Handler.java:958)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:205)
	at android.os.Looper.loop(Looper.java:294)
	at android.app.ActivityThread.main(ActivityThread.java:8177)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
Caused by: kotlin.UninitializedPropertyAccessException: lateinit property handler has not been initialized
	at com.rohit.ble_peripheral.BlePeripheralPlugin$broadcastReceiver$1.onReceive(BlePeripheralPlugin.kt:475)
	at android.app.LoadedApk$ReceiverDispatcher$Args.lambda$getRunnable$0(LoadedApk.java:1802)
	... 10 more
```

So i guess the solution would be to add checks to see if its initialized on the code that catches the state change and if it is, then proceed, if not then don't do anything.

**Comment by cubedude** [comment]
Also, discovered if bluetooth is turned off and trying to initialize BlePeripheral, it will throw an PlatformUnsupportedException that bluetooth is not supported, but if i catch that and then turn on bluetooth after that, it doesn't throw broadcast Intent exception anymore and after bluetooth has been turned on, i can call BlePeripheral.initialize() once more and it works.

**Comment by rohitsangwan01** [comment]
@cubedude thanks for the detailed description, i will check these, meanwhile a PR will be appreciated

**Comment by rohitsangwan01** [comment]
@cubedude should be fixed in version 2.3.1, you can reopen if still exists
